certbot/cli.py

```diff
@@ -750,6 +750,9 @@ def add(self, topics, *args, **kwargs):
 
         """
 
+        if args[0] == '--certbot-route53:auth-propagation-seconds':  # Remove duplicate route53 options from help.
```

Comment:
Why is adding it in the other place insufficient?

Reply:
Adding it to the Apache group isn't ideal.

Found a cleaner way to do this: in `determine_help_topics`, can just always set this topic to false:
```
if chosen_topic == "all":
    return dict([(t, True) if t != 'certbot-route53:auth' else (t, False) for t in self.help_topics])
```

Then it's only one place to keep track of things.

Reply:
Yep, that is certainly better. Have made the commit.